### PR TITLE
OpsCenter 6.8.37 Release

### PR DESCRIPTION
**Pull request by vimal-ds**
----
## Release Notes Automation
If you name your pull-request as "Product x.y.z Release ...", after merging the
PR, a GitHub Action will automatically create a product version tag "product-x.y.z".

Supported product names are:
 * DSE
 * OpsCenter
 * Studio
 * Luna Streaming

Version supports 3 sets or 4 sets of digits.

### CODE REVIEW

**Review by msmygit — APPROVED** [review]
lgtm

**Comment by annieden** [comment]
I recommend ending each sentence in all of the sections with a period. Periods are inconsistently used right now - and should be used everywhere.

**Review by annieden — APPROVED** [review]
Put a period after each bullet's sentence, please!

File: OpsCenter_6.8_Release_Notes.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
 
+# Release Notes for OpsCenter 6.8.37
+14 May 2024
+
+## Backup Service
+* Upgraded Azcopy to v10; added new authentication methods for connecting Azure storage. (OPSC-16022)
+* Upgraded Azure SDK to V12. (OPSC-16414)
+* Fixed an error that could occur when syncing backup meta data from a destination to OpsCenter's backup table. (OPSC-17506)
+* Improved backup activity API and upgraded backup_reports table schema. (OPSC-17511)
+* Added Migration script to upgrade backup_reports table. Set `backup_reports_migrate_data` config to `True` to start migration, reset to `False` after the migration is completed and restart opscenter. (OPSC-17520)
```
Comment (orion104) [inline OpsCenter_6.8_Release_Notes.md:11]: Include the config file and section it is in as well.

**Review by orion104 — APPROVED** [review]

**Pull request merged**